io.gitlab.android_translation_layer.BaseApp.yml

```diff
@@ -0,0 +1,261 @@
+        url: https://android.googlesource.com/platform/external/perfetto.git
+        commit: 93885509be1c9240bc55fa515ceb34811e54a394
+      - type: git
+        dest: third_party/externals/piex
+        url: https://android.googlesource.com/platform/external/piex.git
+        commit: bb217acdca1cc0c16b704669dd6f91a1b509c406
+      - type: git
+        dest: third_party/externals/vulkanmemoryallocator
+        url: https://github.com/GPUOpen-LibrariesAndSDKs/VulkanMemoryAllocator.git
+        commit: 7de5cc00de50e71a3aab22dea52fbb7ff4efceb6
+      - type: git
+        dest: third_party/externals/vulkan-deps
+        url: https://chromium.googlesource.com/vulkan-deps.git
+        commit: 8a3971e5e9c5cddc437d1cf5e4ac1ed0728d6a49
+      - type: git
+        dest: third_party/externals/spirv-cross
+        url: https://github.com/KhronosGroup/spirv-cross.git
+        commit: 12542fc6fc05000e04742daf93892a0b10edbe80
+      - type: git
+        dest: third_party/externals/spirv-headers
+        url: https://github.com/KhronosGroup/spirv-headers.git
+        commit: bdbfd019be6952fd8fa9bd5606a8798a7530c853
+      - type: git
+        dest: third_party/externals/spirv-tools
+        url: https://github.com/KhronosGroup/spirv-tools.git
+        commit: e7c6084fd1d6d6f5ac393e842728d8be309688ca
+      - type: git
+        dest: third_party/externals/vello
+        url: https://github.com/linebender/vello.git
+        commit: ef2630ad9c647b90863cb0915701d54725733968
+      - type: git
+        dest: third_party/externals/vulkan-headers
+        url: https://github.com/KhronosGroup/vulkan-headers.git
+        commit: 9e61870ecbd32514113b467e0a0c46f60ed222c7
+      - type: git
+        dest: third_party/externals/vulkan-tools
+        url: https://github.com/KhronosGroup/vulkan-tools.git
+        commit: 80b010b1e1b08c1f092fb2bfa337faadf8ea1ba3
+      - type: git
+        dest: third_party/externals/wuffs
+        url: https://github.com/google/wuffs-mirror-release-c.git
+        commit: a0041ac0310b3156b963e2f2bea09245f25ec073
+  - name: wolfssl
+    buildsystem: autotools
+    config-opts:
+      - --enable-shared
+      - --disable-opensslall
+      - --disable-opensslextra
+      - --enable-aescbc-length-checks
+      - --enable-curve25519
+      - --enable-ed25519
+      - --enable-ed25519-stream
+      - --enable-oldtls
+      - --enable-base64encode
+      - --enable-tlsx
+      - --enable-scrypt
+      - --disable-examples
+      - --enable-crl
+      - --with-rsa
+      - --enable-certs
+      - --enable-session-certs
+      - --enable-encrypted-keys
+      - --enable-cert-gen
+      - --enable-cert-ext
+      - --enable-clr-monitor
+      - --enable-jni
+    sources:
+      - type: git
+        url: https://github.com/wolfSSL/wolfssl.git
+        tag: v5.7.0-stable
+  - name: vixl
+    buildsystem: meson
+    build-options:
+      config-opts:
+        - -Dsimulator=none
+    only-arches:
+      - aarch64
+      - arm
```

Comment:
```suggestion
```

Reply:
the last runtime that supported it was EOL in 2021. in short it's not supported on flathub

Reply:
applied